src/screens/index.js

```diff
@@ -1,19 +1,39 @@
-import React, { lazy, Suspense } from 'react';
+import React, { lazy, Suspense, useState, useCallback } from 'react';
 import { Router } from '@reach/router';
 
 import { ROUTES } from '@constants/routes';
 import Loading from '@components/Loading';
+import ProtectedRoute, { AuthContext } from '@components/ProtectedRoute';
+
 const Home = lazy(() => import('./Home'));
-const Login = lazy(() => import('./Login'));
+const TimeZones = lazy(() => import('./TimeZones'));
+const NotFound = lazy(() => import('./NotFound'));
 
 function App() {
+  const [isAuth, setIsAuth] = useState(!!localStorage.getItem('isAuth'));
```

Comment:
I'd have this key `isAuth` extratced somewhere. If for some reason you want to change it, there's already 3 places where you'd need to do so.